src/server/mod.rs

```diff
@@ -43,6 +44,7 @@ pub struct Data {
     pub db: Database,
     pub reports_worker: reports::ReportsWorker,
     pub acl: ACL,
+    pub metric: IntCounterVec,
```

Comment:
We'll surely want to add more metrics in the future. What about creating something like:

```rust
pub struct Metrics {
    crater_completed_jobs_total: IntCounterVec,
}
```